Source/Core/DiscIO/WbfsBlob.h

```diff
@@ -19,7 +19,10 @@ class WbfsFileReader : public IBlobReader
 public:
 	static WbfsFileReader* Create(const std::string& filename);
 
-	u64 GetDataSize() const override { return m_size; }
+	// The WBFS format does not save the original file size.
+	// This function returns a constant upper bound
+	// (the size of a double-layer Wii disc).
```

Comment:
It makes some sense to do that, but like you said, we're not going to get a useful hash no matter what we do. Since it's a bit of a guesswork method, I'd say that it shouldn't be included unless it actually is useful.